### ISSUE TITLE
The poseidon contract function is an empty implementation

### ISSUE THREAD

**Issue by ChihChengLiang** [privacy-ethereum/incrementalquintree#1]
### Problem

The Poseidon hash function here is not an interface, they are implementations with an empty function body.

https://github.com/appliedzkp/incrementalquintree/blob/22a68651f0c92ed324d44e67ffb89c9e3b2f506e/contracts/Poseidon.sol#L17

### Solution

Propose to define the interface using Solidity `interface`.

Thank @AbbyLuHui for discovering this issue.

**Comment by weijiekoh** [comment]
Yes, the implementation is empty but the following script generates the necessary EVM bytecode for Poseidon:

https://github.com/appliedzkp/maci/blob/v1/contracts/scripts/compileSol.sh#L21
